sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/DigitalTwinModelsImpl.java

```diff
@@ -105,43 +117,67 @@
         @UnexpectedResponseExceptionType(ErrorResponseException.class)
         Mono<Response<Void>> delete(
                 @HostParam("$host") String host,
+                @HeaderParam("traceparent") String traceparent,
+                @HeaderParam("tracestate") String tracestate,
                 @PathParam("id") String id,
                 @QueryParam("api-version") String apiVersion,
                 Context context);
 
         @Get("{nextLink}")
         @ExpectedResponses({200})
         @UnexpectedResponseExceptionType(ErrorResponseException.class)
-        Mono<Response<PagedModelDataCollection>> listNext(
+        Mono<Response<PagedDigitalTwinsModelDataCollection>> listNext(
                 @PathParam(value = "nextLink", encoded = true) String nextLink,
                 @HostParam("$host") String host,
-                @HeaderParam("x-ms-max-item-count") Integer maxItemCount,
+                @HeaderParam("traceparent") String traceparent,
+                @HeaderParam("tracestate") String tracestate,
+                @HeaderParam("max-items-per-page") Integer maxItemsPerPage,
                 Context context);
     }
 
     /**
-     * Uploads one or more models. When any error occurs, no models are uploaded. Status codes: 200 (OK): Success. 400
-     * (Bad Request): The request is invalid. 409 (Conflict): One or more of the provided models already exist.
+     * Uploads one or more models. When any error occurs, no models are uploaded. Status codes: * 201 Created * 400 Bad
+     * Request * DTDLParserError - The models provided are not valid DTDL. * InvalidArgument - The model id is invalid.
+     * * LimitExceeded - The maximum number of model ids allowed in 'dependenciesFor' has been reached. *
+     * ModelVersionNotSupported - The version of DTDL used is not supported. * 409 Conflict * ModelAlreadyExists - The
+     * model provided already exists.
```

Comment:
Do we call out these error types in our CL? They might be helpful for customers, since they get a list of expected errors. Might be worth adding a link to any documentation that the service team maintains.

Reply:
We don't explicitly call these out in our CL, which does feel odd considering the swagger has that information. I'm not sure what the appropriate way to call this out would be. I'd rather not copy the comments from the PL into the CL since they would get out of date quickly

Reply:
Yeah, maintaining copies is definitely cumbersome. We could ask the service team to upload this info to some MS doc, which we could link from our CL. (if they haven't already). They might not want to maintain this solely in a swagger doc anyway.